environments/prod/prod.tfvars

```diff
@@ -735,6 +735,16 @@ frontends = [
         operator       = "Equals"
         selector       = "_csrf"
       },
+      {
+        match_variable = "RequestHeaderNames"
+        operator       = "Equals"
+        selector       = "x-csrf-token"
+      },      
+      {
+        match_variable = "RequestHeaderNames"
+        operator       = "Equals"
+        selector       = "content-type"
```

Comment:
How is this a problem?

Reply:
It matches sometimes for file uploads ... I am not sure I can post the logs here, but I can show you